assets/src/libraries/BookDetail.jsx

```diff
@@ -140,7 +82,7 @@ export default class BookDetail extends Component {
         <DialogContent className="modal-container">
           <div className="modal-book">
             <div className="modal-book__image-box">
-              {imageUrl}
+              {book.image_url ? <img src={book.image_url} alt="Book cover" className="modal-book__image" /> : null}
```

Comment:
This can be refactored as `{book.image_url && <img src={book.image_url} alt="Book cover" className="modal-book__image" />}`